continuous-pipe.yml.erb

```diff
@@ -0,0 +1,192 @@
+              from_external: true
+
+            ports:
+              - 80
+              - 443
+
+      filter:
+        expression: 'tasks.storage.services.database.created'
+
+  setup:
+    run:
+      image:
+        from_service: web
+
+      commands:
+        - container setup
+
+      environment_variables: &WEB_ENV_VARS
+        APP_USER_LOCAL: false
+        DATABASE_NAME: ${DATABASE_NAME}
+        DATABASE_USER: ${DATABASE_USER}
+        DATABASE_PASSWORD: ${DATABASE_PASSWORD}
+        DATABASE_ROOT_PASSWORD: ${DATABASE_ROOT_PASSWORD}
+        AUTH_HTTP_ENABLED: true
+        AUTH_HTTP_HTPASSWD: ${AUTH_HTTP_HTPASSWD}
+        AUTH_IP_WHITELIST_ENABLED: true
+        AUTH_HTTP_WHITELIST_IPS: ${AUTH_HTTP_WHITELIST_IPS}
+
+    filter:
+      expression: 'tasks.storage.services.database.created and code_reference.branch not in ["production", "uat"]'
+
+  deployment:
+    deploy:
+      services:
+        web:
+          specification:
+            accessibility:
+              from_external: true
+
+            environment_variables:
+              <<: *WEB_ENV_VARS
+
+            resources:
+              requests:
+                cpu: 50m
+                memory: 50Mi
+
+              limits:
+                cpu: 1
+                memory: 2G
+
+            ports:
+              - 80
+              - 443
+
+          deployment_strategy:
+            readiness_probe:
+              type: tcp
+              port: 80
+
+            liveness_probe:
+              type: exec
+              command:
+                - curl
+                - -k
+                - -I
+                - -X GET
+                - -H
+                - "Authorization: Basic ${AUTH_HTTP_AUTHORISATION_STRING}"
+                - https://localhost/
+              initial_delay_seconds: 30
+              period_seconds: 300 # 5 minutes
+              success_threshold: 1
+              failure_threshold: 5
+
+        cron:
+          specification:
+            accessibility:
+              from_cluster: false
+
+            source:
+              from_service: web
+
+            ports: []
+
+            environment_variables:
+              <<: *WEB_ENV_VARS
+              START_MODE: cron
+
+            resources:
+              requests:
+                cpu: 50m
+                memory: 20Mi
+
+              limits:
+                cpu: 1
+                memory: 1G
+
+pipelines:
+  - name: Production
+    tasks:
+      - web_image
+      - storage
+      - reserve_ip
+      - setup
+      - deployment
+    condition: 'code_reference.branch in ["develop", "qa", "staging", "production"]'
+  - name: Pull Requests
+    tasks:
+      - web_image
+      - storage
+      - reserve_ip
+      - setup
+      - deployment
+    condition: '"Ready for review" in pull_request.labels'
+  - name: Remote Development
+    tasks:
+      - web_image
+      - storage
+      - reserve_ip
+      - setup
+      - deployment
+    variables:
+      DEVELOPMENT_MODE: 0
```

Comment:
The PR to the dockerfiles repo will handle 0/1 in a BC way for DEVELOPMENT_MODE and a few other variables, so no need to do it as such.